### ISSUE TITLE
It's recommended to use pigeon to generate platform channel code

### ISSUE THREAD

**Issue by xiaowei-guan** [flutter-webrtc/flutter-webrtc#1300]
https://github.com/flutter/packages/tree/main/packages/pigeon

**Comment by ycherniavskyi** [comment]
Yes, for sure. It was discussed here [What do you think about migrating to Pigeon?](https://github.com/flutter-webrtc/flutter-webrtc/discussions/1011).

**Comment by xiaowei-guan** [comment]
ok

**Comment by cloudwebrtc** [comment]
Sounds great, it looks like C++ is supported

**Comment by cloudwebrtc** [comment]
But I wrote a bridge code from flutter_linux to flutter C++ before 
https://github.com/flutter-webrtc/flutter-webrtc/tree/main/linux/flutter

**Comment by cloudwebrtc** [comment]
but the EventChannel is missing

**Comment by xiaowei-guan** [comment]
I'm trying to generate platfrom channel code by Pigeon for Tizen.
